target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
In U-boot it is in ebu.c when enabling the flash bank 0:

```
 if (ebu_region0_enable) {
                /*
                 * Map EBU region 0 to range 0x10000000-0x13ffffff and enable
                 * region control. This supports up to 32 MiB NOR flash in
                 * bank 0.
                 */
                ltq_writel(&ltq_ebu_regs->addr_sel_0, LTQ_EBU_REGION0_BASE |
                        EBU_ADDRSEL_MASK(ebu_region0_addrsel_mask) | EBU_ADDRSEL_REGEN);
                ltq_writel(&ltq_ebu_regs->con_0, EBU_CON_AGEN_DEMUX |
                        EBU_CON_WAIT_DIS | EBU_CON_PW_16BIT |
                        EBU_CON_ALEC(3) | EBU_CON_BCGEN_INTEL |
                        EBU_CON_WAITWRC(7) | EBU_CON_WAITRDC(3) |
                        EBU_CON_HOLDC(3) | EBU_CON_RECOVC(3) |
                        EBU_CON_CMULT_16);
```
Here is the related declarations for con_0 register:
```

#define EBU_CON_WRDIS                   (1 << 31)
#define EBU_CON_AGEN_DEMUX              (0x0 << 24)
#define EBU_CON_AGEN_MUX                (0x2 << 24)
#define EBU_CON_SETUP                   (1 << 22)
#define EBU_CON_WAIT_DIS                (0x0 << 20)
#define EBU_CON_WAIT_ASYNC              (0x1 << 20)
#define EBU_CON_WAIT_SYNC               (0x2 << 20)
#define EBU_CON_WINV                    (1 << 19)
#define EBU_CON_PW_8BIT                 (0x0 << 16)
#define EBU_CON_PW_16BIT                (0x1 << 16)
#define EBU_CON_ALEC(cycles)            ((cycles & 0x3) << 14)
#define EBU_CON_BCGEN_CS                (0x0 << 12)
#define EBU_CON_BCGEN_INTEL             (0x1 << 12)
#define EBU_CON_BCGEN_MOTOROLA          (0x2 << 12)
#define EBU_CON_WAITWRC(cycles)         ((cycles & 0x7) << 8)
#define EBU_CON_WAITRDC(cycles)         ((cycles & 0x3) << 6)
#define EBU_CON_HOLDC(cycles)           ((cycles & 0x3) << 4)
#define EBU_CON_RECOVC(cycles)          ((cycles & 0x3) << 2)
#define EBU_CON_CMULT_1                 0x0
#define EBU_CON_CMULT_4                 0x1
#define EBU_CON_CMULT_8                 0x2
#define EBU_CON_CMULT_16                0x3
```

In this case this is not the bit 32 but the bit 30, which seems to be ignored and put to 0 by the ltq_writel(&ltq_ebu_regs->con_0) function.